### ISSUE TITLE
I am trying to stop learning when accuracy is 0.8 with callback, but nothing I've tried works.

### ISSUE THREAD

**Issue by dsimeonov140** [SciSharp/Keras.NET#105]
I tried do it with custom callback and the callback runs this python file:

```
import keras

ACCURACY_THRESHOLD = 0.8

class AccHistory(keras.callbacks.Callback):
    def on_batch_end(self, batch, logs={}):
        if(logs.get('acc') > ACCURACY_THRESHOLD):
		self.model.stop_training = True
```
But this row:
`self.model.stop_training = True`
gives me an error.

Then I tried with EarlyStopping:
`var es = new EarlyStopping(monitor: "val_accuracy", mode: "max", baseline: 0.8f, restore_best_weights: false, patience: 0, verbose: 0);`
But this is stops at 0.4-0.6 accuracy.

So how do I do that?

**Comment by deepakkumar1984** [comment]
Able to implement it with this example: https://github.com/SciSharp/Keras.NET/blob/master/Examples/BasicSamples/EarlyStopExample.cs

![image](https://user-images.githubusercontent.com/6328361/77312534-19740080-6d52-11ea-8c3c-f91e7ea9885d.png)

**Comment by dsimeonov140** [comment]
It works now!
This row:
`self.model.stop_training = True`
gives me an error because there was spaces and tabs in python file.

Also I have to change this row:
`if(logs.get('acc') > ACCURACY_THRESHOLD):`
with
`if(logs.get('accuracy') > ACCURACY_THRESHOLD):`
(changed 'acc' with 'accuracy') because it gives me an error: '>' not supported between instances of 'Nonetype' and 'float'
